docs/changelog.rst

```diff
@@ -13,6 +13,8 @@ Fixes:
   and :doc:`/plugins/thumbnails`). :bug:`1448`
 * :doc:`/plugins/permissions`: Fix an error with non-ASCII paths. :bug:`1449`
 * Fix sorting by paths when case-insensitive. :bug:`1451`
+* The rewrite plugin now accepts back-references to matched substrings in the
```

Comment:
The `:doc:` syntax that appears just above lets you link to the plugin in question.